cards/src/main/resources/cards/custom/group0/other_death_knight/koft/minion_hypothermic_outcast.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Hypothermic Outcast",
+  "baseManaCost": 2,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 3,
+  "baseHp": 7,
+  "rarity": "RARE",
+  "description": "Frozen. After this minion takes damage, Silence it.",
+  "trigger": {
+    "eventTrigger": {
+      "class": "DamageReceivedTrigger",
+      "hostTargetType": "IGNORE_OTHER_TARGETS"
+    },
+    "spell": {
+      "class": "SilenceSpell",
+      "target": "SELF"
+    }
+  },
+  "attributes": {
+    "FROZEN": true,
+    "DONT_UNFREEZE": true
```

Comment:
Maybe we can do something simpler here without introducing a new attribute. What if we rewrite the card to say:

`Each turn, freeze this minion. After this minion takes damage, silence it.`

Or use an `"alwaysApply": true` attribute aura.

Reply:
I did it this way because this way actually seemed like the simpler way to me, and also would be less likely to be buggy with effects like Moorabi, and it'd be handy in the future to have a way to interact with the natural unfreezing process